### ISSUE TITLE
Modular State Management: View Actions 25 min

### ISSUE THREAD

**Issue by TimurAsayonok** [TimurAsayonok/point-free-co-tutorial#6]
This is a subtask for main task Modularity https://github.com/TimurAsayonok/point-free-co-tutorial/issues/2
here In need to watch 

- [x] [Introduction](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t6)
- [x] [Transforming a store’s action](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t40)
- [x] [Combining view functions](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t395)
- [x] [Focusing on favorite primes actions](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t525)
- [x] [Extracting our first modular view](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t687)
- [ ] [Focusing on prime modal actions](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t823)
- [ ] [Focusing on counter actions](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t924)
- [ ] [Next time: what’s the point?](https://www.pointfree.co/collections/composable-architecture/modularity/ep74-modular-state-management-view-actions#t1331)

**Comment by TimurAsayonok** [comment]
What we did after **Extracting our first modular view**:

1. we added main `store.view` method that returns Store<LocalValue, LocalAction> and updated our code for using this method
2. we updated favorite primes view for using only local action - `FavoritePrimesAction. it means that this view can work only with its Actions and cannot act with AppActions
3. move `FavoritePrimesView` to FavoritePrime module
